plugins/infinispan100/src/main/java/org/radargun/service/Infinispan100HookedJGroupsTransportLifecycleManager.java

```diff
@@ -0,0 +1,9 @@
+package org.radargun.service;
+
+import org.infinispan.factories.annotations.InfinispanModule;
+import org.infinispan.lifecycle.ModuleLifecycle;
+
+@InfinispanModule(name = "ispn100-radargun-jgroups-transport", requiredModules = "core")
```

Comment:
If you need another component you shouldn't create another module, so I suggest  `name = "ispn100-plugin"` and `Infinispan100PluginModule`